src/schema/v2/me/__tests__/myCollectionUpdateArtworkMutation.test.ts

```diff
@@ -117,7 +166,23 @@ describe("myCollectionUpdateArtworkMutation", () => {
 
       expect(artworkOrError).toEqual({
         artwork: {
+          category: "some strange category",
```

Comment:
could-be-addressed-later:

Can you help me understand the intention of the tests? 

We're mocking the `createArtworkLoader`/`updateArtworkLoader`, so we're not testing that the mutation is actually happening. I _think_ what we're testing is that there is data returned by this method -- though I don't think we're particularly interested in _which_ values are returned, aside from the new `pricePaid.display`. 

Is that to ensure that the `display` property is being resolved? 


Where I'm headed with this -- if we don't care _which_ fields are being returned except the display price, I'd prefer to see fewer fields in this response to make it more obvious which fields we _do_ care about. 

It also might be that we don't care about which fields are returned _including_ the pricePaid - in that case, I don't personally see a value in adding that field to these tests. 

And a final option I can think of is that we care about _all_ the fields being returned -- in which case I'd prefer to see _all_ fields in both this and the create test.

Reply:
I wrote the tests, along with @MounirDhahri to let us test that `display` is working properly,